src/player_team.ts

```diff
@@ -997,7 +997,22 @@ class Team {
       mults[i].base = pings[i].damage;
     }
 
-    let healing = 0;
+    // Apply poison damage.
+    let poison = 0;
+    for (const c of 'pm') {
+      const attr = COLORS.indexOf(c) as Attribute;
+      for (const combo of comboContainer.combos[c]) {
+        let multiplier = 0;
+        if (c == 'p') {
```

Comment:
Change this check to:
combo.attribute == Attribute.POISON

This makes the readability a bit clearer.

Reply:
👍